server/market/balancer.go

```diff
@@ -79,7 +79,7 @@ func NewDEXBalancer(tunnels map[string]PendingAccounter, assets map[uint32]*asse
 			}
 			bb.feeFamily[parentID] = parent.assetInfo
 			for tokenID := range asset.Tokens(parentID) {
-				if tokenID == assetID { // Don't double count
+				if tokenID == assetID || assets[tokenID] == nil { // Don't double count
```

Comment:
Can you comment on the need for this?

Reply:
Yes, that's it. Added a comment unless we should do more.